### ISSUE TITLE
Expose string formatting capability as stand-alone function

### ISSUE THREAD

**Issue by jackirvine97** [brocksam/pyproprop#32]
It would be great to have the ability to directly access `pyproprop`'s string formatting functionality through a stand-alone function as well as through processed properties (as is currently implemented). The desired use case is outlined below:

```
from pyproprop import format_case
my_string = "this IS some-string__that I have etc."
print(format_case(mystring, case="hyphen"))
>>> "this-is-some-string-that-i-have-etc"
````

**Comment by brocksam** [comment]
We've already had the Pyproprop 0.2.2 release. I think as this is adding a module and will result in a rather large feature addition we're better off making this part of Pyproprop 0.3.0.

**Comment by brocksam** [comment]
Addressed in 0.3.0 release.
